app/views/representatives/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class='container'>
+  <% @representatives.each do |representative| %>
+    <div class="row box">
+      <div class=' col-md-4 representative'>
+        <div class='row'>
+          <div class= 'col-md-6'>
+            <b><%= representative.name %></b>
+            <!-- BUG::: <b><%#= representative.candidatures.first.candidate_number %></b> -->
+            <% representative.candidatures.each do |candidature| %>
+              <b><%= candidature.candidate_number %></b>
+            <%end%>
+            <%= link_to (image_tag("politica.png",size: "100x100")),representative_path(representative.id)%>
+            <label><%= representative.occupation %> <%= representative.gender %></label>
+            <label><%= link_to "Listar #{representative.type.pluralize}", sti_representative_path(representative.type) %></label>
+            <div class="dropdown">
+              <button class="btn btn-default dropdown-toggle" type="button" id="dropdownMenu<%= "#{(representative.id)}" %>" data-toggle="dropdown" aria-expanded="true">Ano Eleitoral
+                <span class="caret"></span>
+              </button>
+              <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu<%= "#{(representative.id)}" %>">
+                <% representative.candidatures.each do |candidature| %>
+                  <li role="presentation"><%= link_to candidature.election_year, "#AnoEleição-#{(candidature.id)}", id: candidature.id, :class => "btn" %></li>
+                <%end%>
+              </ul>
+            </div>
+          </div>
+          <% representative.candidatures.each do |candidature| %>
+            <div id='<%= "candidature-#{(candidature.id)}" %>' class= 'col-md-6 hide candidature'>
+              <%= render partial: "candidature", locals: {candidature: candidature} %>
```

Comment:
@isabarros @Bouckaert 

Acho que a gente deveria adotar o padrão: espaços depois de abrir e antes de fechar chaves, exceto quando for interpolar string. Que tal?